javascript/new-src/.jshintrc

```diff
@@ -5,9 +5,9 @@
   "latedef": true,
   "unused": true,
   "noarg": true,
-  "indent": 1,
+  "indent": 2,
   "trailing": true,
-  "white": true,
```

Comment:
I can't recall what i disabled it for...
was it for the white space between ``` function ``` and ``` () ``` ?